src/style.css

```
@@ -1 +1,79 @@
-/* Styles go here */
+body,
+header,
+a,
+img,
+div,
+nav,
+ul,
+li {
+  margin: 0;
+}
+
+body {
```

```suggestion
html {
```
always set the font to the `html { ... }` tag